/usr/local/lib/python2.7/dist-packages/scipy/cluster/vq.py:523: UserWarning: One of the clusters is empty. Re-run kmeans with a different initialization.
  warnings.warn("One of the clusters is empty. "
SubclassesandRetraining.py:136: RuntimeWarning: invalid value encountered in double_scalars
  cosVals[i][j] = np.dot(queryHV, classHV[i][j])/ (np.linalg.norm(queryHV)*np.linalg.norm(classHV[i][j]))
Killed
